docs/notebooks/01_merge.ipynb

```diff
@@ -1,27 +1,505 @@
 {
```

Comment:
<p>I left out individually remapping genes. I think its to much.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>fair enough!</p><p> </p>